pkg/mounter/mounter.go

```diff
@@ -146,17 +143,14 @@ func isCorruptedMnt(err error) bool {
 }
 
 func writePass(pwFileName string, pwFileContent string) error {
-	pwFile, err := os.OpenFile(pwFileName, os.O_RDWR|os.O_CREATE, 0600) // #nosec G304: Value is dynamic
+	pwFile, err := os.OpenFile(pwFileName, os.O_RDWR|os.O_CREATE, 0600)
```

Comment:
Can we add this back `// #nosec G304: Value is dynamic`  as go sec will fail in internal pipeline